administrator-guides/authentication/oauth/keycloak/README.md

```diff
@@ -0,0 +1,50 @@
+# Keycloak
+
+## Configuring Keycloak Identity Provider
+
+Create a client in Keycloak.
+
+1. Provide a client ID: `rocket-chat-client`
+2. Select the client protocol as openid-connect
+3. Select the client access type as confidential
+4. Standard flow implemented: ON
+5. Valid Redirect URL: `http:{Rocket.Chat_ip_address}:port/*`
```

Comment:
```suggestion
5. Valid Redirect URL: `http:{Rocket.Chat_server_address}/*`
```

Mostly just to encourage against setting up while in ip:port form cause will break if they add domain after